entities/TestRunner.js

```diff
@@ -21,7 +21,7 @@ class TestRunner {
   async run() {
     const response = await this.httpRequest();
     const results = this.testConfiguration.checkAssertions(response);
-
+    console.log(results);
```

Comment:
Do you still want this console.log() here?

Reply:
Probably not :)